### PR TITLE
Add default value

### PR DESCRIPTION
**Pull request by Erriez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,9 @@ on:
   uses: devops-actions/action-get-tag@v1.1
   with:
     # Optionally strip `v` prefix
```
Comment (rajbos) [inline README.md]: Let's remove this useless comment now, since it is included in the next line

**Review by rajbos — COMMENTED** [review]
Looks good, thanks for adding it!

Only one small remark

File: README.md (hunk 1 above)
Comment (Erriez) [reply README.md]: Thanks for your review. You're right, fixed in https://github.com/devops-actions/action-get-tag/pull/28/commits/8c9c4fb35c7f2c4532b86ee22a58a0c0e41bf60c.

**Comment by rajbos** [comment]
Thanks for the addition!

**Review by rajbos — APPROVED** [review]

**Pull request merged**